MekHQ/src/mekhq/Utilities.java

```diff
@@ -828,44 +828,76 @@ public static boolean rollProbability(int prob) {
     }
 
     /**
-     * Calculates the age of a character based on their experience level and clan status.
+     * Calculates the age of a pilot based on their experience level and clan status.
      *
-     * @param expLevel the experience level of the character represented by an integer value.
-     * @param clan     a boolean value indicating whether the character is a clan character.
-     * @return the age of the character calculated based on the provided expLevel and clan.
+     * <p>The age is determined by rolling a number of dice (specified by the experience level)
+     * and applying modifications based on clan status. Dice rolls of 6 "explode," meaning an additional
+     * roll is added with one subtracted. If the pilot is from a Clan origin, each roll is halved
+     * (rounded up).</p>
+     *
+     * <p>The results are then added to a base age (18), except for {@code SkillType.EXP_NONE},
+     * where the base age is subtracted at the end.</p>
+     *
+     * <p>Below are the average ages based on the experience level (assuming exploding dice logic):</p>
+     *
+     * <ul>
+     *   <li><b>EXP_NONE</b>:</li>
+     *           <li>- Non-clan: ~25.2 years</li>
+     *           <li>- Clan: ~12.6 years</li>
+     *   <li><b>EXP_GREEN</b>:</li>
+     *           <li>- Non-clan: ~22.2 years</li>
+     *           <li>- Clan: ~20.1 years</li>
```

Comment:
I think the Clan math is a little off. They conduct the Trial of Position around 20, lore establishes Clan Pilots are Veteran, so I would expect regular around 20. Green early likely around teenage.

Reply:
@Hammer, sorry, jumped the gun.  I'll do the review for any follow-ups.